docs/core/v3.0.rst

```diff
@@ -137,6 +138,29 @@ provide a framework for community-defined extensions, which can be
 developed and published independently without requiring centralised
 coordination of all specifications.
 
+See `protocol extensions`_ below.
+
+Stability Policy
```

Comment:
This section is somewhat redundant and also inconsistent with the description regarding "zarr_format".  In particular `zarr_format` claims both backwards and forwards compatibility for a given major version number.

What do you have in mind as far as what type of changes would result in a major, minor, or patch number change?

Reply:
Good catch, I didn't re-check the `zarr_format` contents. I adapted the stability policy now to conform with the `zarr_format` part (no actual semver, but full compatibility across the same major version). There's still two sentences of duplication, not sure if that's fine or too redundant. What do you think about the new version @jbms? If it doesn't provide benefits I'm also not opposed to removing the section again.